README.md

```diff
@@ -154,6 +154,33 @@ Find detailed info on COCO utilities (yolov5 conversion, slicing, subsampling, f
 
 Find detailed info on MOT utilities (ground truth dataset creation, exporting tracker metrics in mot challenge format) at [mot.md](docs/mot.md).
 
+### Input, Export Video Support 
+
+
+A video can be given as input and the results can be exported as video. You can also see the SAHI predictions during inference on the interactive window where fast-forwarding or backwarding is also possible.
+
+One more thing added prediction time to terminal.
+
+For input, export video:
+` sahi predict --model_path yolov5s.pt --model_config_path coco.yaml --model_type yolov5 --export_video --video_path home/burak/video.mp4`   
+
+For view result of prediction during inference:
+` sahi predict --model_path yolov5s.pt --model_config_path coco.yaml --model_type yolov5 --export_video --video_path home/burak/video.mp4  --view_img` 
+
+
+For fast-forwarding or backwarding at result view of prediction during inference::
+
+` sahi predict --model_path yolov5s.pt --model_config_path coco.yaml --model_type yolov5 --export_video --video_path /home/burak/video.mp4  --view_img --forward_backward` 
```

Comment:
update the argument `--forward_backward` with `--fast_forward_backward`

Reply:
When I think as Turkish, fast-forward is not enough. If fast-forward is enough in English, we can say fast-forward. I'm not very good at English.

Reply:
Yes, fast-forward is sufficient in English to cover both meaning.